specs/eip4844/validator.md

```diff
@@ -79,27 +78,30 @@ def validate_blobs_and_kzg_commitments(execution_payload: ExecutionPayload,
 
 3. If valid, set `block.body.blob_kzg_commitments = blob_kzg_commitments`.
 
-#### Constructing the `SignedBeaconBlockAndBlobsSidecar`
-To construct a `SignedBeaconBlockAndBlobsSidecar`, a `signed_beacon_block_and_blobs_sidecar` is defined with the necessary context for block and sidecar proposal.
-
-##### Block
-Set `signed_beacon_block_and_blobs_sidecar.beacon_block = block` where `block` is obtained above.
+#### Constructing the `SignedBlobSidecar`
+To construct a `SignedBlobSidecar`, a `signed_blob_sidecar` is defined with the necessary context for block and sidecar proposal.
 
 ##### Sidecar
-Coupled with block, the corresponding blobs are packaged into a sidecar object for distribution to the network.
 
-Set `signed_beacon_block_and_blobs_sidecar.blobs_sidecar = sidecar` where `sidecar` is obtained from:
+Coupled with block, the corresponding blobs are packaged into sidecar objects for distribution to the network.
```

Comment:
These are not coupled!

```suggestion
The corresponding blobs are packaged into sidecar objects for distribution to the network.
```

Reply:
ffc78e99282e65cc4da92c4dba0743be434d367b